src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep.java

```diff
@@ -295,7 +300,27 @@ private void perform(Run<?,?> build, Launcher launcher, TaskListener listener) t
     	final IProgress progress = new ScanProgress(listener);
     	final boolean suspend = m_wait;
     	final IScan scan = ScanFactory.createScan(getScanProperties(build, listener), progress, m_authProvider);
-        
+		JenkinsAuthenticationProvider checkAppScan360Connection = new JenkinsAuthenticationProvider(m_credentials,build.getParent().getParent());
+        if(m_type.equals("Dynamic Analyzer") && checkAppScan360Connection.isAppScan360()){
+			throw new AbortException(Messages.error_dynamic_analyzer_AppScan360());
+		}
+
+        if(!checkAppScan360Connection.isAppScan360()){
+            if(m_authProvider.getacceptInvalidCerts()){
+                progress.setStatus(new Message(1,Messages.warning_asoc_certificates()));
+            }
+        }
+
+		if(m_intervention && checkAppScan360Connection.isAppScan360()){
+			progress.setStatus(new Message(1,Messages.warning_allow_intervention_AppScan360()));
+		}
+
+		if(getScanProperties(build, listener).get("openSourceOnly") !=null){
+			if(getScanProperties(build, listener).get("openSourceOnly").equals("") && checkAppScan360Connection.isAppScan360()){
```

Comment:
Is the presence of the "openSourceOnly" property indicative of this being an open source only scan? If so, we shouldn't need to check the value of the property.

Reply:
Okay, Removed the unnecessary check.